repos/system_upgrade/el7toel8/actors/repositoriesblacklist/libraries/library.py

```diff
@@ -0,0 +1,38 @@
+from leapp.libraries.stdlib import api
+from leapp.models import  RepositoriesBlacklisted, RepositoriesFacts, RepositoriesMap
+
+
+def get_list_of_optional_repos():
+    """
+        Returns a dictionary with these fields:
+            key = from_id
+            value = to_id
+    """
```

Comment:
Yes please std indentation as it is used in other docstrings. Additionally, this is not valid doc.
```
    """
    Return a dict of optional repositories based on RepositoriesMap: { 'from_id' : 'to_id'}
    
    It consumes RepositoriesMap messages and create map (dict) of optional repositories
    on RHEL 7 system to CRB repositories on RHEL 8. See the RepositoriesMap model.. 
    """
```